packages/veritone-redux-common/src/helpers/api/index.js

```diff
@@ -25,5 +25,6 @@ export function getCredentialsMode() {
   // app.veritone.com on port 80 in prod, or
   // ie. local.veritone.com on any port in dev
   const isVeritoneInternalApp = origin.match(/\.veritone\.com(:\d{1,5})?$/);
-  return isVeritoneInternalApp ? 'include' : 'omit';
+  const isAiwareAnywhereInternalApp = origin.match(/\.aiware\.com(:\d{1,5})?$/);
+  return (isVeritoneInternalApp || isAiwareAnywhereInternalApp) ? 'include' : 'omit';
```

Comment:
Let's remove this logic to support all domains. Please update the change log, and also update other packages depending on veritone-redux-common, and do major release.